src/phases/query/CollScanPredicateSelectivity.yml

```diff
@@ -0,0 +1,427 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This workload tests the performance of conjunctive collection scan queries where the order of
```

Comment:
What are the important metrics? Can you add this information to the description?

Reply:
OperationThroughput would probably be the most important, but we look at all the latency measurements as well. They will become increasingly important once plan-cache is enabled, since maxlatency/latency99thpercentile will probably be the only place where optimization time (without hitting the plan cache) is tracked until the optimizer metrics project (and relevant perf-infra work) completes.

Reply:
Good points, I didn't take plan cache into account. I updated the description.